CMakeLists.txt

```diff
@@ -864,6 +864,13 @@ if(BUILD_TESTS)
     )
     target_link_libraries(base64_test PRIVATE ${CMAKE_THREAD_LIBS_INIT})
 
+    add_unit_test(
+      cose_test ${CMAKE_CURRENT_SOURCE_DIR}/src/crypto/test/cose.cpp
+    )
+    target_link_libraries(cose_test PRIVATE ${CMAKE_THREAD_LIBS_INIT})
```

Comment:
Can this be one call to `target_link_libraries` ?

Reply:
It can